package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "resin-lint --typescript src tests",
     "prettify": "prettier --config ./node_modules/resin-lint/config/.prettierrc --write \"{src,tests}/**/*.ts\" gulpfile.js",
     "test": "gulp test",
+    "prebuild": "rimraf build/",
```

Comment:
Belts and braces? I think that the reason why the problem was not detected in some of my testing of PR #23 is that the `build/src` subfolder was copied on top of the exiting `build` folder that still had `js` files from previous test runs.